packages/quantic/force-app/main/default/lwc/quanticTooltip/quanticTooltip.js

```diff
@@ -30,23 +38,127 @@ export default class QuanticTooltip extends LightningElement {
    * Method tha hides the tooltip.
    */
   @api hideTooltip() {
+    this.resetPosition();
     this.isVisible = false;
   }
 
+  /** @type {boolean} */
+  isInitialLoad = true;
+  /** @type {NodeJS.Timeout} */
+  resizeTimer;
+  /** @type {boolean} */
+  displayTooltipAboveTarget = true;
   /** @type {boolean} */
   isVisible = false;
 
+  connectedCallback() {
+    window.addEventListener('resize', this.handleResizeAndScroll);
+    window.addEventListener('scroll', this.handleResizeAndScroll);
+  }
+
+  renderedCallback() {
+    if (this.isInitialLoad) {
+      this.isInitialLoad = false;
+      this.updateTooltipMaxWidth();
+      this.updateTooltipVerticalPosition();
+    }
+  }
+
+  disconnectedCallback() {
+    window.removeEventListener('resize', this.handleResizeAndScroll);
+    window.removeEventListener('scroll', this.handleResizeAndScroll);
+  }
+
+  handleResizeAndScroll = () => {
+    clearTimeout(this.resizeTimer);
+    // eslint-disable-next-line @lwc/lwc/no-async-operation
+    this.resizeTimer = setTimeout(() => {
+      this.updateTooltipMaxWidth();
+      this.updateTooltipVerticalPosition();
+    }, 300);
+  };
+
+  updateTooltipMaxWidth() {
+    const windowWidth = window.innerWidth;
+    const styles = this.template.host?.style;
+    styles.setProperty('--adapted-max-width', `${windowWidth}px`);
```

Comment:
Have you tried it on the hosted insight panel when the tab isn't active?
It feels like we're going to have the same issues than with the tab bar, the tooltip won't have it's proper width set because the innerWidth will be 0 for the non-active tab

Reply:
will give it a try and see 👍